src/test/java/expediaproductobjects/FilterOptions.java

```diff
@@ -0,0 +1,26 @@
+package expediaproductobjects;
+
+import org.openqa.selenium.WebDriver;
+
+public class FilterOptions extends BasePage{
+	
+	public String starRatingXpath = "//label[@for='f-star-rating-[RATING]']";
+	public String slider = "//div[@role='slider']";
+
+
+	public FilterOptions(WebDriver driver) {
+		super(driver);
+	}
+	
+	public void starRating(int desiredRating) {
+		
+		String rating = String.valueOf(desiredRating);
+		this.click(starRatingXpath.replace("[RATING]", rating));
+	}
+	
+	public void nightlyPrice (int minPrice, int maxPrice) {
+		String lowPrice = String.valueOf(minPrice);
+		String highPrice = String.valueOf(maxPrice);
+		this.click(slider);
+	}
```

Comment:
I don't understand how this method works. To me it seems like it just accepts a max/min and then clicks the slider, doing nothing. It doesn't seem like the click has anything to do with the min and max. Is this a working method?

Reply:
I am still working on this method. Didn't figure out yet how to control the sliding bar for the price range.
I mentioned this in the PR description that this method isn't completed yet and im still working on it.

Reply:
Oh ok. Don't send a PR for this though. Point of a PR is to push to the live version right, but if you pushed this into the live version it's gonna cause bugs or be redundant at best because its not functioning code

Reply:
Fine ill delete this method and commit again

Reply:
Just take your time completing the task and then send a PR with the completed code